src/main/java/com/myproject/todo/todo/model/Todo.java

```diff
@@ -0,0 +1,45 @@
+package com.myproject.todo.todo.model;
+
+import java.util.Date;
+
+public class Todo {
+	
+	private Integer id;
+	private Date date; 
+	private String task;
+	public Integer getId() {
+		return id;
+	}
+	public void setId(int id) {
+		this.id = id;
+	}
+	public Date getDate() {
+		return date;
+	}
+	public void setDate(Date date) {
+		this.date = date;
+	}
+	public String getTask() {
+		return task;
+	}
+	public void setTask(String task) {
+		this.task = task;
+	}
+	
+	protected Todo()
+	{
+		
+	}
+	
+	public Todo(int id, Date date, String task) {
```

Comment:
why int and not integer?

Reply:
So that It doesnt crash when it is shifted to a database

Reply:
how exactly it is going to crash if it's in integer?